code/style.css

```diff
@@ -1,5 +1,107 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Archivo&family=Cabin&family=Encode+Sans&display=swap');
 
 body {
-  background-color: lightblue;
+  background-color: rgb(240, 239, 239);
+  font-family: 'Encode Sans', sans-serif;
+  margin: 0;
 }
+
+h1 {
+  margin: 0;
+  text-align: center;
+  color: rgb(240, 239, 239);
+  font-size: 70px;
+}
+
+h2 {
+  margin: 0;
+  text-align: center;
+  color: rgb(240, 239, 239);
+  font-size: 50px;
+}
+
+h3 {
+  margin: 0;
+  text-align: center;
+  color: #654321;
+  font-size: 30px;
+}
+
+h4 {
+  Color: black;
+}
+
+header {
+  background-image: url(img/NissyenArts_Bandphoto_Guitar.png);
+  background-size: cover;
+  min-height: 300px;
+  background-position: center;
+  padding-top: 150px;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 5px;
+}
+
+.icons {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  padding: 30px;
+  align-items:center;
+}
+
+.icon {
+  padding: 10px;
+}
+
+.icon:hover {
+    -moz-box-shadow: 0 0 10px #ccc;
+    -webkit-box-shadow: 0 0 10px #ccc;
+    box-shadow: 0 0 10px #ccc;
+}
+
+.arrow {
+  padding-left: 10px;
+  padding-right: 10px;
+  width: 30px;
+}
+
+symbol {
+  padding: 0, 10px;
+}
+
+section {
+  padding: 30px;
+  padding-top: 0;
+}
+
+span {
+  color:#654321;
+}
+
+.logo {
+  width: 150px;
+  align:center;
+  margin-right: 150px;
+}
+
+ul {
+  padding-left: 25%;
+  }
+
+  li {
+  padding-top: 10px;
+  }
```

Comment:
This should not be indented, it should be at the same level as the other elements